mix.exs

```diff
@@ -1,6 +1,6 @@
 # Copyright(c) 2015-2023 ACCESS CO., LTD. All rights reserved.
 
-antikythera_dep = {:antikythera, [github: "access-company/antikythera", ref: "e65018a857a9dd334019fb6d6808e886e239ae27"]}
+antikythera_dep = {:antikythera, [github: "access-company/antikythera", ref: "634063688c631d7006418441b87be6032a3ae16a"]}
```

Comment:
The contents are almost the same, but this one would be correct.

```suggestion
antikythera_dep = {:antikythera, [github: "access-company/antikythera", ref: "4790839059b17763030d77488da532c2a4b6e30a"]}
```

Reply:
Thank you.
Updated to the current reviewing commit https://github.com/access-company/antikythera_instance_example/pull/26/commits/d530e190d8d602e8f19f5d698e1a94448fd545f1